src/esbmc/CMakeLists.txt

```diff
@@ -45,6 +45,6 @@ message("Looking at ${CVC_LINK_DIR}")
 target_link_libraries(esbmc ${OLD_FRONTEND_TARGETS} ${SOLIDITY_FRONTEND_TARGETS} ${PYTHON_FRONTEND_TARGETS}
                             ${GOTO_CONTRACTOR_TARGETS} ${JIMPLE_FRONTEND_TARGETS} ${JIMPLE_FRONTEND_TARGETS}
                             clangcfrontend clangcppfrontend filesystem symex pointeranalysis langapi util_esbmc bigint
-                            solvers clibs gotoalgorithms cache ${Boost_LIBRARIES} goto2c)
+                            solvers clibs gotoalgorithms cache ${Boost_LIBRARIES} goto2c m stdc++)
```

Comment:
libm should only be added for Unix targets.

Reply:
OK, I've created a new variable that gets set differently for each OS in OSConfiguration. It's called `OS_INCLUDE_LIBS`, it gets used on targets that did not compile successfully. It's set as a blank value for any OS other than UNIX now.